src/main/java/RPN.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * LCW MCON 264
+ * Evaluates an RPN expression
+ * by: Miriam Dreyfuss
+ */
+
+import java.util.Stack;
+
+public class RPN
+{
+    public String evaluate(String expression)
+    {
+        String[] values = expression.split(" ");
+        Stack<Double> stack = new Stack<>();
+        for (String character: values)
+        {
+            try
+            {
+                stack.push(Double.parseDouble(character));
+            }
+            catch (NumberFormatException notInt)
```

Comment:
I'd say you should only use Exceptions for error cases, NOT for valid input like operators.